### PR TITLE
Bug 1999225: Update 4.9 CSV description with new profiles and options

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

File: manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -68,6 +71,7 @@ spec:
       In addition to the profiles, the following parameters can be configured:
 
       * `deschedulingIntervalSeconds` - Set the number of seconds between descheduler runs. A value of `0` in this field runs the descheduler once and exits.
+      * `profileCustomizations` - Allows certain profile parameters to be tweaked, such as PodLifetime
```
Comment (bergerhoffer) [inline manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml:74]: * Needs a period at the end
* Should this be "PodLifeTime" (T capitalized) as in the strategy name, or should it be "podLifetime" as in the actual field in the YAML?

Comment (damemi) [reply manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml]: Updated to match the yaml field, and point to the readme for more info

File: manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -68,6 +71,7 @@ spec:
       In addition to the profiles, the following parameters can be configured:
 
       * `deschedulingIntervalSeconds` - Set the number of seconds between descheduler runs. A value of `0` in this field runs the descheduler once and exits.
+      * `profileCustomizations` - Allows certain profile parameters to be tweaked, such as `podLifetime` (see README for more info).
```
Comment (ingvagabund) [inline manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml:74]: README -> [README](https://github.com/openshift/cluster-kube-descheduler-operator#profile-customizations)

Comment (damemi) [reply manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml]: Updated

**Pull request merged**